sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueListenerTests.cs

```diff
@@ -37,6 +37,7 @@ public class QueueListenerTests : LiveTestBase<WebJobsTestEnvironment>
         private const string TestFunctionId = "TestFunction";
         private Mock<QueueClient> _mockQueue;
         private QueueListener _listener;
+        private QueueScaleMonitor _scaleMonitor;
```

Comment:
Do we want also add tests for `_targetScaler` which is part of `QueueListener` now?

Reply:
added tests specifically for target scaler in another file